extensions/2.0/Khronos/KHR_xmp/schema/glTF.KHR_xmp.schema.json

```diff
@@ -0,0 +1,20 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title" : "KHR_xmp glTF extension",
+    "type": "object",
+    "description": "Metadata about the glTF asset.",
+    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+    "properties": {
+        "packets": {
```

Comment:
To be aligned with other glTF arrays, `packets` should not be empty, i.e., it should have `"minItems": 1`.

Reply:
Thanks